controllers/apply/lib/joi-extensions/phone-number.test.js

```diff
@@ -20,10 +20,10 @@ test('return error for invalid phone numbers', () => {
 
 test('format valid phone numbers', () => {
     const validUkPhoneNumbers = [
```

Comment:
Updated references in test files even though they are not public facing just to avoid any confusion.